mailbox/cassandra/src/test/java/org/apache/james/mailbox/cassandra/CassandraSubscriptionManagerTest.java

```diff
@@ -59,22 +61,31 @@
 public class CassandraSubscriptionManagerTest extends AbstractSubscriptionManagerTest {
 
     @ClassRule public static DockerCassandraRule cassandraServer = new DockerCassandraRule();
-    
-    private CassandraCluster cassandra;
 
-    @Before
-    public void init() {
+    private static CassandraCluster cassandra;
+
+    @BeforeClass
+    public static void setUpClass() {
         CassandraModuleComposite modules = new CassandraModuleComposite(
             new CassandraSubscriptionModule(),
             new CassandraMailboxCounterModule());
-        cassandra = CassandraCluster.create(modules, cassandraServer.getHost());
+        cassandra = CassandraCluster.create(modules, cassandraServer.getIp(), cassandraServer.getBindingPort());
```

Comment:
I don't understand why in this commit sometimes you replace by getHost, sometimes you remove getHost call.
That's a good example of a commit you should split. This is already a big commit, and you should not mix the initial commit goal with some little improvements like that.